python/main.py

```diff
@@ -92,24 +127,31 @@ async def add_item(
     with open(image_path, "wb") as f:
         f.write(image_data)
 
-    insert_item(Item(name=name, category=category, image_name=image_name))
+    category_id = get_category_id(db, category)
+
+
+    insert_item(Item(name=name, category_id=category_id, image_name=image_name),db)
```

Comment:
```suggestion
    insert_item(Item(name=name, category_id=category_id, image_name=image_name), db)
```

細かい修正ですが、スペース入れておくと可読性あがります。